<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_profile</class>
 <widget class="QWidget" name="add_profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>URL Spletne Strani</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ime Spletne Strani</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Ime Profila</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txt_profile_name"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txt_page_title"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txt_page_url">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txt_profile_id"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_delete">
         <property name="text">
          <string>Izbri≈°i</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_save">
         <property name="text">
          <string>Vnesi</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txt_profile_id</tabstop>
  <tabstop>txt_profile_name</tabstop>
  <tabstop>txt_page_title</tabstop>
  <tabstop>txt_page_url</tabstop>
  <tabstop>btn_save</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
